packages/vscode-extension/src/extension.ts

```diff
@@ -63,11 +63,6 @@ export function deactivate(context: ExtensionContext): undefined {
 export async function activate(context: ExtensionContext) {
   handleUncaughtErrors();
 
-  if (Platform.OS !== "macos" && Platform.OS !== "windows") {
```

Comment:
can you throw an error here after sending a popup message instead of throwing it in the platform utility? 

you could change te default to "unsupported" or something like that. I don't like the idea of utility that seems to be a static value crashing the application